tuxemon/core/components/save.py

```diff
@@ -123,42 +129,96 @@ def load(slot):
     if not os.path.exists(save_path):
         return
 
-    saveFile = shelve.open(save_path)
     saveData = dict()
+    try:
+        with open(save_path, 'r') as save_file:
+            json_data = json.load(save_file)
+            tempinv = dict()
+
+            for slug,quant in json_data['inventory'].items():
+                tempinv1 = dict()
+                tempinv1['item'] = Item(slug)
+                tempinv1['quantity'] = quant
+                tempinv[tempinv1['item'].name] = tempinv1
+            saveData['inventory'] = tempinv
+
+            tempmon = list()
+            for mon in json_data['monsters']:
+                tempmon1 = Monster()
+                tempmon1.load_from_db(mon['slug'])
+                tempmon1.load_from_savefile(mon);
+                tempmon.append(tempmon1)
+            saveData['monsters'] = tempmon
+
+            # TODO: unify loading and game instancing
+            # Loop through the storage item keys and re-add the surface.
+            tempstorage = dict()
+            for keys, values in json_data['storage'].items():
+                if keys == 'items':
+                    tempinv = dict()
+
+                    for slug,quant in values.items():
+                        tempinv1 = dict()
+                        tempinv1['item'] = Item(slug)
+                        tempinv1['quantity'] = quant
+                        tempinv[tempinv1['item'].name] = tempinv1
```

Comment:
The name attribute is dependent on the locale set.  Is there a way to avoid its use here?